programs/bridge-program/src/lib.rs

```diff
@@ -21,11 +21,19 @@ pub mod bridge_program {
     pub fn initialize_contract(
         ctx: Context<InitializeContract>,
         relayer_pk: Pubkey,
```

Comment:
Small note: relayer_pk should be renamed to relayer_pbk to make it clear it's the public key